neo4j/test-integration/examples_test.go

```diff
@@ -540,21 +518,11 @@ func createDriverWithCustomizedConnectionPool(uri, username, password string) (n
 	return neo4j.NewDriver(uri, neo4j.BasicAuth(username, password, ""), func(config *neo4j.Config) {
 		config.MaxConnectionLifetime = 30 * time.Minute
 		config.MaxConnectionPoolSize = 50
-		config.ConnectionAcquisitionTimeout = 2 * time.Minute
 	})
 }
 
 // end::config-connection-pool[]
 
-// tag::config-connection-timeout[]
```

Comment:
not sure if that's OK to just remove the entire tag

Reply:
If the driver manual tries to pull this in, you'd also have to adjust that (remove the reference from there).